src/searchengine/nova3/helpers.py

```diff
@@ -59,7 +59,7 @@ def getBrowserUserAgent() -> str:
     return f"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:{nowVersion}.0) Gecko/20100101 Firefox/{nowVersion}.0"
 
 
-headers: Dict[str, Any] = {'User-Agent': getBrowserUserAgent()}
+headers: dict[str, Any] = {'User-Agent': getBrowserUserAgent()}
```

Comment:
Could you explain this change?

Reply:
typing.Dict is deprecated since python 3.9.

Reply:
> typing.Dict is deprecated since python 3.9. This is migrating it away.

Actually, I was wondering what is the difference between `Dict` and `dict` (except for the case of the first character).

Reply:
https://docs.python.org/3/library/typing.html#typing.Dict